generator/src/lib.rs

```diff
@@ -2464,15 +2468,144 @@ pub fn write_source_code(path: &Path) {
         #bitflags_macro
         #handle_nondispatchable_macro
         #define_handle_macro
+        // #(#feature_code)*
+        // #(#definition_code)*
+        // #(#enum_code)*
+        // #(#bitflags_code)*
+        // #(#constants_code)*
+        // #(#extension_code)*
+        // #feature_extensions_code
+        // #const_debugs
+        // #(#aliases)*
```

Comment:
Dead code?

Reply:
I initially left it there for reference and then forgot about it. I've removed it and then also moved all of the file writes down below into a single section to make it more clear what files are being written. 

I also rebased in the `const` bitflags fix that was merged earlier. Oddly enough I ran into that today too and was going to put in another PR!